opentelemetry/src/test/scala/zio/telemetry/opentelemetry/TracingTest.scala

```diff
@@ -73,6 +71,37 @@ object TracingTest extends DefaultRunnableSpec {
               )
             )
         },
+        testM("scopedSpan") {
+          for {
+            tracer <- ZIO.service[Tracer]
+            _ <- UIO.effectTotal {
+                  val span = tracer.getCurrentSpan
+                  span.addEvent("In legacy code")
+                  Thread.sleep(10)
```

Comment:
How about testing if the thread local is available here?

Reply:
Maybe? I'm still trying to determine if this is even a viable approach... I assume you're thinking along the lines of `assert(span != null) && assert(span.name == "Scoped")`?

Reply:
I've added an assertion that the current context != the root context, which seems to work pretty well in that if you swap `scopedSpan` for `span` the test blows up.